src/crypto/tls/tls_cf.go

```diff
@@ -7,6 +7,7 @@ import (
 	circlPki "github.com/cloudflare/circl/pki"
 	circlSign "github.com/cloudflare/circl/sign"
 	"github.com/cloudflare/circl/sign/eddilithium3"
+	"github.com/cloudflare/circl/sign/eddilithium2"
```

Comment:
nits: order package import? I wonder if go fmt does that

Reply:
Good point. Fixed.